test/flycheck-test.el

```diff
@@ -3044,14 +3044,26 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
 
 (flycheck-ert-def-checker-test erlang erlang error
   (flycheck-ert-should-syntax-check
-   "language/erlang/error.erl" 'erlang-mode
+   "language/erlang/erlang/error.erl" 'erlang-mode
    '(7 nil error "head mismatch" :checker erlang)))
 
 (flycheck-ert-def-checker-test erlang erlang warning
   (flycheck-ert-should-syntax-check
-   "language/erlang/warning.erl" 'erlang-mode
+   "language/erlang/erlang/warning.erl" 'erlang-mode
    '(6 nil warning "wrong number of arguments in format call" :checker erlang)))
 
+(flycheck-ert-def-checker-test erlang-rebar3 erlang error
+  (flycheck-ert-should-syntax-check
+   "language/erlang/rebar3/src/erlang-error.erl" 'erlang-mode
+   '(6 nil warning "wrong number of arguments in format call" :checker erlang-rebar3)
+   '(7 nil error "head mismatch" :checker erlang-rebar3)))
+
+(flycheck-ert-def-checker-test erlang-rebar3 erlang warning
+  (flycheck-ert-should-syntax-check
+   "language/erlang/rebar3/src/erlang-warning.erl" 'erlang-mode
+   '(6 nil warning "wrong number of arguments in format call" :checker erlang-rebar3)
+   '(7 nil error "head mismatch" :checker erlang-rebar3)))
```

Comment:
Sorry, I wasn't sure what the expected value should be. The failures that are a problem return `nil`.